app/controllers/ems_common.rb

```diff
@@ -561,6 +563,10 @@ def button
     end
   end
 
+  def provider_documentation_url
+    "http://manageiq.org/documentation/getting-started/#adding-a-provider"
```

Comment:
meant EMS controller*. (e.g defining it again in EmsContainerController will take effect over the container provider new/edit documentation link)